add a strict note about the compilers run-time (#766)
conda-forge/conda-forge.github.io: src/user/tipsandtricks.rst

In addition to the channel priority we recommend to always install your packages inside a new environment instead the root environment from anaconda/miniconda.
Using envs make it easier to debug problems with packages and ensure the stability of your root env.

> Note: in the past ``conda-forge`` used to vendorize some of ``defaults`` dependencies that were not built in our infrastructure,

Reviewer: Can we put this in a `..note::` box for consistency?

Author: Done. Thanks!

like compilers run-times, to avoid the mixing channel problem.

> However, with the ``strict`` option, we no longer have to vendorize those (which has its own problems too),

Reviewer: Do we now recommend everybody to use strict? If so we probably should also put that here: http://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge

Reviewer 2: Yes, I agree.

Author: In `conda 4.7` where `strict` will be default and we will need to update that but I guess that, b/c mixing channels without `strict` is broken, we should add it there too.

Author: Done! Not sure if we should leave the comment about conda 4.7 there though.

> However, with the ``strict`` option, we no longer have to vendorize those (which has its own problems too),

Reviewer: I personally would leave out `(which has its own problems too)` but I guess that's a matter of taste.

Reviewer 2: ```suggestion
However, with the ``strict`` option, we no longer have to vendorize those (this led to its own set of problems),
```
@croth1 it seems like a good idea to indicate why these channels are marked as `broken` and effectively not available.  My preference would be for some brief explanation of what those problems where but it seems good to at least indicate that there was a good reason not to keep them available, given that it's caused some inconvenience for people like me that where told to use only the `conda-forge` channel before the `strict` option was available.

Author: Maybe a better explanation for this should be in the `News` section?

Reviewer 2: Sure, that's fine.
